src/ray/common/ray_config_def.h

```diff
@@ -612,6 +612,10 @@ RAY_CONFIG(bool, scheduler_avoid_gpu_nodes, true)
 /// Whether to skip running local GC in runtime env.
 RAY_CONFIG(bool, runtime_env_skip_local_gc, false)
 
+/// The ray cluster id. Different cluster should have unique ID.
+/// This fields is used to isolate data stored in DB.
+RAY_CONFIG(std::string, cluster_id, "default")
```

Comment:
Isn't the default supposed to be an unique id if the different cluster should have unique id?